packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -264,7 +264,9 @@ export interface IncrementalSegmentAction<TContext> {
 export interface IncrementalBlockAction<TContext> {
 	(state: IncrementalMapState<TContext>);
 }
-
+/**
+ * @deprecated - unused and will be removed
```

Comment:
This change causes it to now become unused. Is that a safe breaking change to make in main?

Reply:
yes. merge tree isn't exported so not subject to breaking changes. this is now just an unused interface. i'm actually being extra safe by deprecating and not just directly removing.